apps/admin/backend/package.json

```diff
@@ -49,6 +49,7 @@
     "@votingworks/utils": "workspace:*",
     "better-sqlite3": "^7.5.3",
     "compression": "^1.7.4",
+    "csv-stringify": "^6.4.0",
```

Comment:
@benadida had commented on an original PR for the CSV exports that we should probably be using a package to handle all the CSV formatting rather than trying to do it ourselves, so here I introduce the most common and supported one.